Tigran Kirakosov/style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* Highier level containers states */
+.weather-container_main {
+  padding: 80px 110px;

```

Comment:
the same about ordering

Reply:
I missed something. What was that previous note about ordering?...

Reply:
yeap - it's about css properties